README.rst

```diff
@@ -163,11 +163,19 @@ To disable an existing MFE, remove the corresponding entry from the ``MFE_APPS``
 Using custom translations to your MFEs
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-By default, translations are pulled from the `openedx/openedx-translations repository <https://github.com/openedx/openedx-translations>`_ on build time.
+During docker image build, this plugin runs ``make pull_translations`` for each Micro-frontend. This
+program is used by the ``Dockerfile`` to pull translations from the `openedx/openedx-translations repository <https://github.com/openedx/openedx-translations>`_ via the `openedx-atlas <https://github.com/openedx/openedx-atlas>`_.
+
+The ``make pull_translations`` command passes the ``ATLAS_OPTIONS`` environment variable to the ``atlas pull`` command. This allows specifying a custom repository or branch to pull translations from:
 
 To use your own translations repository, you can override the
 ``ATLAS_REPOSITORY: your-organization/translations`` configuration variable.
 
+If a custom branch is needed, you can override the
+``ATLAS_REVISON: your-branch`` configuration variable.
```

Comment:
For clarity, the values for ATLAS_REPOSITORY, ATLAS_REVISION, and ATLAS_OPTIONS are not provided by default. They are only needed for overriding the default pipeline, correct?

Reply:
@DawoudSheraz I think the docs weren't clear. I've just pushed an update. Does that answer your question?